assets/_sass/_navbar.scss

```diff
@@ -151,14 +150,23 @@ $nav-item-border: darken($navbar-default-bg, 8%);
 
       .brand-name-grid {
         display: flex;
+        padding: 0 0px 0px 10px;
+        flex-direction: column;
         justify-content: center;
-        width: 100%;
+        align-items: flex-start;
 
         .brand-name {
-          color: black;
-          margin: 0;
-          font-weight: bold;
-          text-align: center;
+          margin-bottom: 2px;
+          font-size: 14px;
+          font-style: normal;
+          font-weight: 600;
+          line-height: 19px;
```

Comment:
Why are we changing this? We just removed it due to feedback in the last apple release

Reply:
I'm not aware, the same was done previously. I was making changes to just show this label and clean up the look. I think this is not exactly final since the designs are not finalised as yet. That conversation is still open. What would you suggest? @CharlRitterDev

Reply:
Apple wanted it larger, so it was changed to inherit their h4 typography, which is 18px.
They also wanted it bold & centred

From the last apple deploy, I'd say none of the font related stuff here is needed

Reply:
The parent class also needs to stay at 100%, because they specifically wanted it centered.

Reply:
Sorry Mpilo, the designs do not quite reflect the latest changes. Thanks @CharlRitterDev for catching. We should only add the label and it should be centered. Let's not touch the title or logo area.  @CharlRitterDev please help Mpilo since you did the last update to make sure we don't have regression.